src/actionTypes.js

```diff
@@ -341,35 +341,37 @@ type EventMessageDeleteAction = {|
   type: typeof EVENT_MESSAGE_DELETE,
   messageIds: number[],
 |};
+type TopicLinks = $ReadOnlyArray<{| +text: string, +url: string |}> | $ReadOnlyArray<string>;
+// This is current to feature level 109.
 type EventUpdateMessageAction = {|
```

Comment:
nit: blank line between these definitions

(Not sure why there wasn't one between the existing two; seems like an oversight.)